test/EasyPost/UserTest.php

```diff
@@ -49,6 +49,7 @@ public function testCreate()
 
         $this->assertInstanceOf('\EasyPost\User', $user);
         $this->assertStringMatchesFormat('user_%s', $user->id);
+        $this->assertEquals($user->name, 'Test User');
```

Comment:
Order is wrong here.
```suggestion
        $this->assertEquals('Test User', $user->name);
```